### PR TITLE
feat(tokens-page): add collator and properly order tokens

### PR DESCRIPTION
**Pull request by zahnster**
* Re-orders the design tokens in the formatters to be alphanumeric (uses collator function previously in the Design Tokens List code in the Docs site)

### CODE REVIEW

File: packages/paste-website/src/components/tokens-list/index.tsx [hunk 1]
```diff
@@ -97,17 +99,26 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
...
-              ))}
+              {tokens[tokenCategory]
+                .sort((a, b) => {
+                  if (tokenCategory === 'font-weights') {
+                    const aValue: string = typeof a.value === 'string' ? a.value : a.value.toString();
+                    const bValue: string = typeof b.value === 'string' ? b.value : b.value.toString();
+                    return collator.compare(aValue, bValue);
+                  }
+                  return collator.compare(a.name, b.name);
+                })
```
Comment (TheSisb) [inline packages/paste-website/src/components/tokens-list/index.tsx:110]: question: just wondering if we can run this once in an effect and save it to state or if this needs to run every re-render

Comment (zahnster) [reply packages/paste-website/src/components/tokens-list/index.tsx:110]: Alternate thought: what if we did this sorting on the design-token formatters level? I was tempted to do it there first but then saw the prior art.

Comment (TheSisb) [reply packages/paste-website/src/components/tokens-list/index.tsx:110]: You can try that for the gatsby formatter export. If it doesn't work or is difficult to implement, you can alternatively consider a constant file somewhere in the website package that imports the theme, sorts it, and exports it out for this page to use.

**Review by SiTaggart — APPROVED** [review]
Seems reasonable to me

**Pull request merged**